include/clang/Serialization/ASTBitCodes.h

```diff
@@ -1322,7 +1322,6 @@ namespace clang {
       EXPR_OBJC_ARRAY_LITERAL,
       EXPR_OBJC_DICTIONARY_LITERAL,
 
-    
       /// \brief An ObjCEncodeExpr record.
```

Comment:
It looks like there are only white space changes to this file.  Please revert the white space changes.   It will make updating to newer versions of the clang sources more work because we'll have to examine these diffs.